system/ee/ExpressionEngine/Addons/pro_search/libraries/Pro_search_filters.php

```diff
@@ -102,16 +102,15 @@ public function __construct()
                 }
 
                 // Compose file name
-                $file = $dir . "/lsf.{$item}.php";
```

Comment:
Unfortunately, I'm not sure if we can remove support for lsf files right now. I think it will most likely break any existing site that uses them. We probably need to check for the existence of `lsf.{$item}.php` and `psf.{$item}.php`. This is why I just left it as lsf, since it would allow everything existing to just work.